spec/models/about_spec.rb

```diff
@@ -12,7 +12,10 @@ def register_stat(name, stats_block)
     )
   end
 
-  after { DiscoursePluginRegistry.reset! }
+  after do
+    About.clear_stats_cache
```

Comment:
Instead of having to clear the cache manually, you can just use [use_redis_snapshotting](https://github.com/discourse/discourse/blob/95b09dd77702b49ea08d99f30ee5ac94053d7079/spec/helpers/redis_snapshot_helper.rb#L4)

```
Rspec.describe About do
  use_redis_snapshotting
end
```